order-book/src/main/java/com/market/banica/order/book/service/grpc/AuroraClient.java

```diff
@@ -81,12 +81,17 @@ private void startTrackingItems(List<String> trackedItems, String clientId) thro
             Context.CancellableContext withCancellation = Context.current().withCancellation();
             cancellableStubs.put(product, withCancellation);
             itemMarket.addTrackedItem(product);
-            withCancellation.run(() -> startMarketStream(request));
+
+            try {
+                withCancellation.run(() -> startMarketStream(request, product));
+            } catch (Exception e) {
+                withCancellation.cancel(e);
```

Comment:
log with an error could be added here

Reply:
Added the error logging, sorry for the delay. :)